client/src/components/FeedbackPage/FeedbackFormSection/FeedbackFormSection.tsx

```diff
@@ -190,39 +53,18 @@ const FeedbackFormSection: FunctionComponent = () => {
       [inputName]: {
         ...feedbackForm[inputName],
         value: inputValue,
-        valid: isValid,
-        errMsg: message
+        valid,
+        errMsg
       }
     });
   };
 
-  const createUpdatedForm = (
-    validatedInputs: { valid: boolean; errMsg: string }[],
-    inputNames: FeedbackFormInput[],
-    form: FeedbackForm
-  ) =>
-    validatedInputs.reduce(
-      (prevFields, currField, currentIndex) => ({
-        ...prevFields,
-        [inputNames[currentIndex]]: {
-          ...form[inputNames[currentIndex]],
-          ...currField
-        }
-      }),
-      {}
-    );
-
   const handleFormChange = () => {
-    const inputNames = Object.keys(FeedbackFormInput).map(key => key as FeedbackFormInput);
+    const inputNames = Object.keys(FeedbackFormInput) as FeedbackFormInput[];
 
-    const validatedInputs = feedbackFormInputs.map((input, i) => {
-      const { isValid: valid, message: errMsg } = inputValidator(
-        input.value!,
-        inputNames[i],
-        feedbackFormConfig[inputNames[i]].validation
-      );
-      return { valid, errMsg };
-    });
+    const validatedInputs = feedbackFormInputs.map((input, i) =>
+      inputValidator(input.value, inputNames[i], feedbackFormConfig[inputNames[i]].validation)
+    );
 
     const updatedForm = createUpdatedForm(validatedInputs, inputNames, feedbackForm);
```

Comment:
`inputNames` variable is a constant that won't be updated, please move it to utils and use in `createUpdatedForm` function without passing as argument